src/gui/src/components/queries/saved-item.tsx

```diff
@@ -172,8 +178,15 @@ const SavedQueryItem = ({
             <p className="text-sm w-[500px] truncate">{editQuery}</p>
           )}
         </div>
-        <div className="flex col-span-2 items-center gap-2">
-          <p className="text-sm truncate">{item.context}</p>
+        <div className="flex col-span-2 items-center gap-2 w-full">
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger className="w-full text-left">
+                <p className="text-sm truncate">{item.context}</p>
+              </TooltipTrigger>
```

Comment:
```suggestion
              <TooltipTrigger className="text-sm truncate w-full text-left">
                {item.context}
              </TooltipTrigger>
```

I think we can do this same thing all on the `<TooltipTrigger>`. It's one less element and less error prone since the `truncate` on the `<p>` class doesn't always work depending on how the parent gets rendered. That's what happened in #337.